homeassistant/remote.py

```diff
@@ -523,3 +523,17 @@ def call_service(api, domain, service, service_data=None):
 
     except HomeAssistantError:
         _LOGGER.exception("Error calling service")
+
+
+def get_config(api):
```

Comment:
I know. Was just added to test the configuration output. If you are OK with this addition, I will add tests.